Purchases/SwiftSources/LocalReceiptParsing/Builders/InAppPurchaseBuilder.swift

```diff
@@ -0,0 +1,98 @@
+//
+// Created by Andrés Boedo on 7/29/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+struct InAppPurchaseBuilder {
+    private let containerBuilder: ASN1ContainerBuilder
+    private let dateFormatter: ISO3601DateFormatter
+
+    init() {
+        self.containerBuilder = ASN1ContainerBuilder()
+        self.dateFormatter = ISO3601DateFormatter.shared
+    }
+
+    func build(fromContainer container: ASN1Container) throws -> InAppPurchase {
```

Comment:
this is very similar to the `AppleReceiptBuilder` in concept. Same comments apply